### ISSUE TITLE
Remove ci/kokoro/readme once no longer needed

### ISSUE THREAD

**Issue by devjgm** [googleapis/google-cloud-cpp#3522]
This issue is blocked on  #3621 and #3620

**Comment by coryan** [comment]
@devjgm this depends on deciding what our README instructions will say, if they say "Use Bazel" we need to change these builds to actually test that, getting the builds to work fast is not that useful.

Feel free to remove the assignment once we have decided about the README.

**Comment by devjgm** [comment]
Related to https://github.com/googleapis/google-cloud-cpp/issues/3621 and https://github.com/googleapis/google-cloud-cpp/issues/3620

Once those are finished, we shouldn't need the remaining `ci/kokoro/readme` builds.

Should we change this issue (or file a new one) to remove those builds?

**Comment by coryan** [comment]
Changing this issue sounds good.

**Comment by devjgm** [comment]
Done. removed myself as I'm not actively working on this one yet.
